src/service_args.py

```diff
@@ -161,132 +123,4 @@ def __customize_service_args(self):
-        chain_json_path = f"{c.CHAIN_SPEC_PATH}/tinker-raw.json"
-
-        utils.download_chain_spec(chain_json_url, 'tinker-raw.json')
-        self.__set_chain_name(chain_json_path, 0)
-
-    def __clover(self):
-        self.__set_chain_name(Path(c.HOME_PATH, 'specs/clover-para-raw.json'), 0)
-
-    def __polkadex(self):
-        self.__set_chain_name(Path(c.HOME_PATH, 'polkadot-parachain-raw.json'), 0)
-
-    def __polkadex_mainnet(self):
-        self.__set_chain_name(Path(c.HOME_PATH, 'customSpecRaw.json'), 0)
-
-    def __unique(self):
-        if self.chain_name == 'unique':
-            chain_json_url = 'https://raw.githubusercontent.com/UniqueNetwork/unique-chain/master/chain-specs/unique.json'
-        elif self.chain_name == 'quartz':
-            chain_json_url = 'https://raw.githubusercontent.com/UniqueNetwork/unique-chain/master/chain-specs/quartz.json'
-        else:
-            raise ValueError("Unsupported chain name.")
-
-        chain_json_path = f'{c.CHAIN_SPEC_PATH}/{self.chain_name}-raw.json'
-
-        utils.download_chain_spec(chain_json_url, f'{self.chain_name}-raw.json')
-        self.__set_chain_name(chain_json_path, 0)
-
-    def __crust(self):
-        if self.chain_name == 'crust-mainnet':
-            self.__set_chain_name('mainnet', 0)
-        elif self.chain_name == 'crust-maxwell':
-            self.__set_chain_name('maxwell', 0)
-        elif self.chain_name == 'crust-rocky':
-            self.__set_chain_name('rocky', 0)
-
-    def __origintrail(self):
-        if exists(c.BINARY_PATH):
-            chain_json_url = f'https://raw.githubusercontent.com/OriginTrail/origintrail-parachain/v{utils.get_binary_version()}/res/origintrail-parachain-2043-raw.json'
-        else:
-            chain_json_url = 'https://raw.githubusercontent.com/OriginTrail/origintrail-parachain/develop/res/origintrail-parachain-2043-raw.json'
-        utils.download_chain_spec(chain_json_url, f'{self.chain_name}-raw.json')
-        chain_json_path = f'{c.CHAIN_SPEC_PATH}/{self.chain_name}-raw.json'
-        self.__set_chain_name(chain_json_path, 0)
-
-    def __asset_hub_rococo(self):
-        chain_json_url = 'https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/cumulus/parachains/chain-specs/asset-hub-rococo.json'
-        chain_json_path = f'{c.CHAIN_SPEC_PATH}/{self.chain_name}.json'
-        utils.download_chain_spec(chain_json_url, f'{self.chain_name}.json')
-        self.__set_chain_name(chain_json_path, 0)
-
-    def __liberland(self):
-        chain_json_url = 'https://raw.githubusercontent.com/liberland/liberland_substrate/main/substrate/specs/mainnet.json'
-        chain_json_path = f'{c.CHAIN_SPEC_PATH}/{self.chain_name}.json'
-        utils.download_chain_spec(chain_json_url, f'{self.chain_name}.json')
-        self.__set_chain_name(chain_json_path, 0)
+            self.__add_firstchain_args(['--wasm-runtime-overrides', c.WASM_PATH])
```

Comment:
Why remove empty line at end of file?

Reply:
I was adding missing `)` in the end and removed the empty line as well. Anyhow it's fixed now!